dev-portal/scripts/deploy-stack.js

```diff
@@ -2,80 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0
 'use strict'
 
-const { execute, r, red, green } = require('./utils.js')
+const { red, green } = require('./utils.js')
 
 const deployerConfig = require('./get-deployer-config.js')
 const writeConfig = require('./write-config.js')
-
-const errors = []
-
-function getRequired (key) {
-  const value = deployerConfig[key]
-  if (value) return value
-  errors.push(key + ' must be defined')
-}
-
-function getOptional (key, orElse) {
-  return deployerConfig[key] || orElse
+const deployTemplate = require('./deploy-template.js')
```

Comment:
It's a little confusing that you call this `deployTemplate` when you use it, but it's defined as `deployerConfig` in the file.

Reply:
`deployerConfig` is the argument - it's an arrow function. 😉

Also, the deployer config includes other things like profiles and such.

Reply:
So it is. I would much prefer it being a named function of some variety so people like me don't make that mistake and get confused again. :p